database/seeders/AdminRolePermissionsSeeder.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+use Spatie\Permission\Models\Role;
+use Spatie\Permission\Models\Permission;
+
+class AdminRolePermissionsSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $permissions = Permission::all();
```

Comment:
dont create new file for admin and sub admin just create one seeder as role seeder and one as permission seeder

Reply:
Role & Permission seeder are already created these seeders were created to seed permissions for a role instead of directly putting them in database. Should I seed permissions for role one by one in the role seeder?